api/handlers/find_user.go

```diff
@@ -48,3 +28,13 @@ func (f *findUser) Handle(fup users.FindUsersParams) middleware.Responder {
 	return res
 
 }
+
+func filteredMap(fileteredName *string) map[string]interface{} {
```

Comment:
Instead of passing only the name 
pass, the whole params object.
it will help in the future if added more filters to the API.

Reply:
Done. Kindly Check.

Reply:
where?